src/PhpSpec/Loader/StreamWrapper.php

```diff
@@ -88,4 +88,9 @@ public function stream_eof(): bool
     {
         return feof($this->fileResource);
     }
+
+    public function stream_set_option ( int $option , int $arg1 , int $arg2 ) : bool
```

Comment:
```suggestion
    public function stream_set_option(int $option, int $arg1, int $arg2): bool
```

Reply:
Oh I missed that one. I definitely agree.